osgar/drivers/oak_camera.py

```diff
@@ -55,12 +68,15 @@ def __init__(self, config, bus):
         assert self.flood_light_current <= 1500, self.flood_light_current  # The limit is 1500 mA.
         self.is_color = config.get('is_color', False)
         self.video_encoder = get_video_encoder(config.get('video_encoder', 'mjpeg'))
+        self.is_stereo_images = config.get('is_stereo_images', False)
+        if self.is_stereo_images:
+            assert self.is_depth, self.is_depth
```

Comment:
why is this linked stereo -> depth??

Reply:
I started with easier way where the stereo_images is an extension of depth. Yes, it could be independent as well. I will consider it.

Reply:
The depth and stereo images can be enabled independently now.